tests/test_watcher_extended.py

```
@@ -0,0 +1,577 @@
+
+            result = watcher_with_mocks.fetch_rss()
+            assert result is None
+
+
+class TestAsyncJobAcceptance:
+    """Tests for async job acceptance wrapper."""
+
+    def test_async_job_acceptance_wrapper_success(self, watcher_with_mocks):
+        """Test successful job acceptance via wrapper."""
+
+        async def mock_accept(job_data):
```

Ruff (0.14.14)

Unused function argument: `job_data`

(ARG001)